cmd/pets/dry-run.go

```diff
@@ -6,26 +6,19 @@ import (
 
```

Comment:
Does dry-run.go need to be removed permanently? it doesn't work the same way as it used to. i think it is just a leftover and could be deleted?